velox/docs/develop/types.rst

```diff
@@ -155,6 +156,18 @@ used in IPADDRESS/IPPREFIX related functions. This type can be used to
 create IPPREFIX networks as well as to check IPADDRESS validity within
 IPPREFIX networks.
 
+IPPREFIX represents an IPV6 or IPV4 formatted IPV6 address along with a one byte
+prefix length. Its physical type is VARBINARY but has a fixed length of 17 bytes.
+The format that the address is stored in is defined as part of `(RFC 4291#section-2.5.5.2) <https://datatracker.ietf.org/doc/html/rfc4291.html#section-2.5.5.2>`_
+The prefix length is stored in the last byte of the VARBINARY array.
+The IP address stored is the canonical(smallest) IP address in the
+subnet range. This type can be used in IP subnet functions.
+
+Example::
+
+   IPPREFIX 'FFFF:FFFF:FFFF:FFFF:FFFF:FFFF:FFFF:FFFF/32' -- IPPREFIX 'FFFF:FFFF:0000:0000:0000:0000:0000:0000/32'
```

Comment:
Is the left hand side the original data input and the right hand side representing what is actually stored based on "The IP address stored is the canonical(smallest) IP address in the
subnet range? Perhaps the example could be a bit more clear on that?

Reply:
Added explanation in example section.